config/toggles.js

```diff
@@ -92,12 +94,12 @@ const longTermToggles = {
     'queueStats'),
   enableRealtimeActivityLogs: envVarIncludes(pe, 'ENABLE_ACTIVITY_LOGS',
     'realtime'),
+  enableSigtermActivityLog: envVarIncludes(pe, 'ENABLE_ACTIVITY_LOGS',
```

Comment:
stuff like this should always go in alphabetical order

Reply:
Dont know I you mean here... Sig is after Real - int that right? The file doesn't seem to follow this convention. enableClock is in between Worker and Redis.

Reply:
yeah, but all the "activity log" entries are grouped together alphabetically

Reply:
it's fine